packages/dashmate/configs/system/base.js

```diff
@@ -96,7 +98,7 @@ module.exports = {
     dapi: {
       envoy: {
         docker: {
-          image: 'dashpay/envoy:0.24-dev',
+          image: `dashpay/envoy:${version}`,
```

Comment:
We shouldn't do that because `dashmate update` won't work to fetch minor updates. Versions must be `0.x` or `0.x-{pre-release tag}`. You can use `semver` library to get this information from the version.